ext/kv/remote.rs

```diff
@@ -543,12 +580,37 @@ async fn call_remote<
       .post(&full_url)
       .header("x-transaction-domain-id", metadata.database_id.to_string())
       .header("authorization", format!("Bearer {}", metadata.token))
+      .header("x-watch", if streaming { "1" } else { "0" })
```

Comment:
Can't this be part of the protobuf?

Reply:
This controls the *shape* of the response (streaming vs oneoff) rather than the *content*, and we want middleboxes (if any) to be able to look at it.